minecraft_servers/skydinse/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+    "server_name": "skydinse",
+    "nice_name": "Skydinse",
+    "direct_ip": "skydinse.de",
+    "server_wildcards": [
+      "%.skydinse.net",
+      "%.skydinse.de",
+      "%.skydinse.tk"
+    ],
+    "supported_languages": [
+      "de",
+      "en"
+    ],
+    "social": {
+        "web": "https://skydinse.de",
+        "web_shop": "https://shop.skydinse.de",

```

Comment:
Shop is not live - Please only link pages that are available.

Reply:
Removed shop-link